diesel_cli/tests/print_schema.rs

```diff
@@ -115,6 +115,12 @@ fn print_schema_custom_types() {
     );
 }
 
+#[test]
+#[cfg(feature = "postgres")]
```

Comment:
Any reason not to run these tests on sqlite and mysql?

Reply:
Made Sqlite and MySQL variants and removed the cfg attribute.

I wrote another test which checks that schema names are included appropriately with `rust_name` table names as well, tying that to PostgreSQL in the same way as the `print_schema_specifying_schema_name` test. Let me know if this is appropriate. 🙂